### PR TITLE
Trouble with scores and polymorphic associations

### PR DESCRIPTION
**Pull request by rlconley**

### CODE REVIEW

**Comment by rlconley** [comment]
Spent a lot of time trying to figure out session and controller tests.  Never got it to work.  Commented code is some things I tried.  Also created a problem by refactoring some of Clinton & Vance's code by accident.  Found and fixed all those.  Took time but got a lot of experience reading that code.  Views are only placeholders.  Left off at testing questions.  Discouraged by the lack of product, but I actually think I learned a lot.  Coder at Ruby Meetup was very helpful in breaking down what I am supposed to do to created controller tests, but that was already Sun night.

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ class QuestionsController < ApplicationController
+    def show
+      if question do
```
Comment (cndreisbach) [inline app/controllers/questions_controller.rb]: `if` doesn't require a `do`.

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,37 @@
 Rails.application.routes.draw do
+  get 'answers/create'
```
Comment (cndreisbach) [inline config/routes.rb]: You can remove all these `get` routes.

**Comment by cndreisbach** [comment]
Good progress so far.

**Pull request closed without merging**